.swift-format

```diff
@@ -0,0 +1,11 @@
+{
+    "version": 1,
+    "lineLength": 10000,
+    "indentation": {
+        "spaces": 4
+    },
+    "lineBreakBeforeEachArgument": true,
```

Comment:
This rule feels a little too opinionated to me, not sure if anyone else has an opinion

Reply:
I think this option helps make the diff ever-so-slightly cleaner to read.

Reply:
This would result in a pretty huge diff. While I agree it can enhance readability in some cases, I'm not sure we should make the jump right away. I'd omit this for now and maybe we should have a wider discussion first.